### ISSUE TITLE
output commandline parameter broken

### ISSUE THREAD

**Issue by alfbr** [OPM/opm-simulators#1499]
As the title say, the output command line parameter has no effect. I have not done a bisection, but it worked as of February 5th, and is broken as of June 12th.

**Comment by andlaus** [comment]
how can I reproduce this? is the 2018.04 release also affected?

**Comment by alfbr** [comment]
Just consult the flow manual for usage description.

**Comment by andlaus** [comment]
can't you be a bit more concrete? I tried `flow output=false $DECK_FILE`, this did indeed produce plenty of terminal output. I patched the simulator side, but the ECL output code and the well model still won't shut up. (I tried to fix the well model, but the `WellSwitchingLogger` code is too entangled for me to fix it up with a drive-by patch.) I doubt that the situation was substantially different ever after the `flow` binary has become a hydra-simulator...
